cli/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "msadoc-cli",
```

Comment:
Should we call the package `msadoc-cli` or `@msadoc/cli`?

Reply:
The second pls.